internal/service/federatedsettingsorgconfig/data_source_federated_settings_connected_org.go

```diff
@@ -153,15 +152,15 @@ func dataSourceMongoDBAtlasFederatedSettingsOrganizationConfigRead(ctx context.C
 		return diag.FromErr(fmt.Errorf("error setting `post_auth_role_grants` for federatedSettings IdentityProviders: %s", err))
 	}
 
-	if err := d.Set("role_mappings", federatedsettingsidentityprovider.FlattenRoleMappings(federatedSettingsConnectedOrganization.RoleMappings)); err != nil {
+	if err := d.Set("role_mappings", FlattenRoleMappings(federatedSettingsConnectedOrganization.RoleMappings)); err != nil {
```

Comment:
don't know if it applies here but in general it's better to use Get() like GetRoleMappings in SDK types

Reply:
This change was only because I moved some methods to federated_settings_connected_org, where it was used. Don't want to include further changes to this if possible